### DISCUSSION TITLE
Every time wrong "HomeConnectError: insufficient scope: Insufficient

### DISCUSSION THREAD

**Discussion by Donni1966** [alexkn/node-red-contrib-homeconnect#65] [Q&A]
I try to get a status from my cofeemaschine to create a dashboard on my internal Webpage. So if i test the API all fine, get infos and so on. So i create this: 
![Bildschirmfoto 2022-06-06 um 11 55 06](https://user-images.githubusercontent.com/61660957/172139833-ccf0de29-b05e-4031-9299-d23215e05f48.png)
and i try it with coffeemaker-monitor an the rest one also. 
![Bildschirmfoto 2022-06-06 um 11 54 56](https://user-images.githubusercontent.com/61660957/172139893-ac117db0-64fc-4945-87fb-087990965154.png)
than this ... 

and if i send a message to the node i get this - what ever i try? 
![Bildschirmfoto 2022-06-06 um 11 54 50](https://user-images.githubusercontent.com/61660957/172139984-ba166671-48bc-4613-9e5d-480140aa1f7c.png)
i give up ... what is my fault?

**Comment by Donni1966** [comment]
find a a way. let scope empty!

**Reply by nicedevil007** [reply]
can you tell us how exactly you fixed it?
I already tryed it without any scope, but it didn't work.

I'm using `IdentifyAppliance`

**Reply by Donni1966** [reply]
not realy 
may be this helps you?
<img width="602" alt="Bildschirmfoto 2022-07-26 um 22 07 18" src="https://user-images.githubusercontent.com/61660957/181102995-cb54ef9a-de69-4f56-9631-b13041147bcf.png">
